borg/helpers.py

```diff
@@ -669,7 +671,10 @@ def decorated_function(*args):
 @memoize
 def uid2user(uid, default=None):
     try:
-        return pwd.getpwuid(uid).pw_name
+        if sys.platform != 'win32':
+            return pwd.getpwuid(uid).pw_name
+        else:
+            return os.getlogin()
```

Comment:
On Windows it probably doesn't matter too much, since UID/GID/perms are almost meaningless.

To support proper permissions it would make sense to add a platform_windows.pyx module that pulls the SD or ACLs (there are two) and can also set it by GUID / by user name (which would be similar to --numerical-ids).